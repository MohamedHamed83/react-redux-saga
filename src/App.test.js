import React from 'react';
import ReactDOM from 'react-dom';
 import renderer from 'react-test-renderer'
import App from './App';

describe('app',()=>{
  it('should render the app',()=>{
    const tree= renderer.create(<App />)
     expect(tree.toJSON()).toMatchSnapshot()
 })
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

})
