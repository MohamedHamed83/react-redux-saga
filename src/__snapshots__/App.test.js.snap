// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app should render the app 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h1
      className="App-title"
    >
         
    </h1>
  </header>
  <p
    className="App-intro"
  />
</div>
`;
